### PR TITLE
Flattened legal terms-and-policies section

### LINKED ISSUE #4197: Flatten legal pages - terms and policies

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Flattened legal terms-and-policies section
- Updated navigation
- Updated redirects
- Added some special redirects requested by legal team

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- look at the files in the /legal directory and compare to this [spreadsheet](https://docs.google.com/spreadsheets/d/12kuuGgwRqN3k7wlRc4QKIuZifjvxbynizO_LEjKjBq4/edit#gid=0)
* test the link [/legal/terms](http://0.0.0.0:8001/legal/terms) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/terms) redirects also test that [/terms](http://0.0.0.0:8001/terms) redirects as well
* test the link [/legal/intellectual-propery-policy](http://0.0.0.0:8001/legal/intellectual-propery-policy) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/intellectual-property-policy) redirects also test that [/ip-policy](http://0.0.0.0:8001/ip-policy) redirects as well
* test the link [/legal/intellectual-property-policy/2013-05-14](http://0.0.0.0:8001/legal/intellectual-property-policy/2013-05-14) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/intellectual-property-policy/2013-05-14) redirects
* test the link [/legal/online-account-terms](http://0.0.0.0:8001/legal/online-account-terms) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/online-account-terms) redirects
* test the link [/legal/terms-of-service](http://0.0.0.0:8001/legal/terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/terms-of-service) redirects
* test the link [/legal/confidentiality-agreement](http://0.0.0.0:8001/legal/confidentiality-agreement) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/confidentiality-agreement) redirects
* test the link [/legal/developer-terms-and-conditions](http://0.0.0.0:8001/legal/developer-terms-and-conditions) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/developer-terms-and-conditions) redirects
* test the link [/legal/launchpad-terms-of-service](http://0.0.0.0:8001/legal/launchpad-terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/launchpad-terms-of-service) redirects
* test the link [/legal/livepatch-terms-of-service](http://0.0.0.0:8001/legal/livepatch-terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/livepatch-terms-of-service) redirects
* test the link [/legal/jaas-beta-terms-of-service](http://0.0.0.0:8001/legal/jaas-beta-terms-of-service) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/jaas-beta-terms-of-service) redirects
* test the link [/legal/systems-information-notice](http://0.0.0.0:8001/legal/systems-information-notice) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/systems-information-notice) redirects
* test the link [/legal/font-licence](http://0.0.0.0:8001/legal/font-licence) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/font-licence) redirects
* test the link [/legal/privacy-policy](http://0.0.0.0:8001/legal/privacy-policy) and [the old link](http://0.0.0.0:8001/legal/terms-and-policies/privacy-policy) redirects also test that [/privacy-policy](http://0.0.0.0:8001/privacy-policy) redirects as well

## Issue / Card

Fixes #4197

### CODE REVIEW

File: templates/legal/developer-terms-and-conditions/2016-04-06.html [hunk 1]
```diff
@@ -60,7 +60,7 @@ <h3 id="dev-terms-support">6. Support</h3>
...
+      <p>Canonical may make logos or phrases available to you to publicise your App&#39;s availability through the Client Software. Canonical hereby grants you a worldwide, non-exclusive licence during the Term to use such logos or phrases for the purpose of marketing your App in connection with its availability through the the Client Software, subject to Canonical&#39;s Intellectual Property Rights policy, which Canonical may update from time to time at <a href="https://www.ubuntu.com/legal/intellectual-property-policy">www.ubuntu.com/legal/intellectual-property-policy</a>.</p>
```
Comment (anthonydillon) [inline templates/legal/developer-terms-and-conditions/2016-04-06.html]: This link should be relative.

File: templates/legal/developer-terms-and-conditions/index.html [hunk 2]
```diff
@@ -78,7 +78,7 @@ <h3 id="dev-terms-trial-versions">7. Trial versions</h3>
         the trial version, then the commercial terms in this Agreement apply.</p>
       <h3 id="dev-terms-marketing">8. Marketing</h3>
       <p>Canonical may make logos or phrases available to you to publicise your App&apos;s availability through the Client Software. Canonical hereby grants you a worldwide, non-exclusive licence during the Term to use such logos or phrases for the purpose
-        of marketing your App in connection with its availability through the the Client Software, subject to Canonical&apos;s Intellectual Property Rights policy, which Canonical may update from time to time at <a href="https://www.ubuntu.com/legal/terms-and-policies/intellectual-property-policy">www.ubuntu.com/legal/terms-and-policies/intellectual-property-policy</a>.</p>
+        of marketing your App in connection with its availability through the the Client Software, subject to Canonical&apos;s Intellectual Property Rights policy, which Canonical may update from time to time at <a href="https://www.ubuntu.com/legal/intellectual-property-policy">www.ubuntu.com/legal/intellectual-property-policy</a>.</p>
```
Comment (anthonydillon) [inline templates/legal/developer-terms-and-conditions/index.html]: These link should be relative.

File: templates/legal/service-terms/2015-09-09.html [hunk 3]
```diff
@@ -38,7 +38,7 @@ <h2>Intellectual Property Rights</h2>
...
             <li class="p-list__item">Ubuntu is licensed under applicable Open Source Software licences and not this Agreement. If Canonical creates or provides any software to Customer in the course of providing the Services, Canonical will provide such software under an Open Source Software licence, unless otherwise licensed under this Agreement.</li>
-            <li class="p-list__item">During and after the term of this Agreement, Customer will comply with Canonical's Intellectual Property Rights policy at <a href="https://www.ubuntu.com/legal/terms-and-policies/intellectual-property-policy">https://www.ubuntu.com/legal/terms-and-policies/intellectual-property-policy</a>.</li>
+            <li class="p-list__item">During and after the term of this Agreement, Customer will comply with Canonical's Intellectual Property Rights policy at <a href="https://www.ubuntu.com/legal/intellectual-property-policy">https://www.ubuntu.com/legal/intellectual-property-policy</a>.</li>
```
Comment (anthonydillon) [inline templates/legal/service-terms/2015-09-09.html]: This link should be relative

**Comment by anthonydillon** [comment]
There is a mix of dataprivacy and data-privacy. Could we try and align them.

**Comment by pmahnke** [comment]
closing as there are non-associated commits in here

**Pull request closed without merging**